demo/background.js

```diff
@@ -1,5 +1,5 @@
 import chromeP from 'webext-polyfill-kinda';
-import addDomainPermissionToggle from '..';
```

Comment:
Good stuff.

Figured out in https://github.com/fregante/webext-events/pull/9/files#diff-9c2553967c8a2b4dfa83f16ffeb500742fcb2488ab343e5f0c16b17013c0adad